state/futures/actions.ts

```diff
@@ -1635,6 +1635,68 @@ const submitCMTransferTransaction = async (
 	}
 };
 
+export const updateStopLossAndTakeProfit = createAsyncThunk<void, void, ThunkConfig>(
+	'futures/updateStopLossAndTakeProfit',
+	async (_, { getState, dispatch, extra: { sdk } }) => {
+		const marketInfo = selectMarketInfo(getState());
+		const account = selectCrossMarginAccount(getState());
+		const wallet = selectWallet(getState());
+		const tradeInputs = selectCrossMarginTradeInputs(getState());
+		const marginDelta = selectCrossMarginMarginDelta(getState());
+		const desiredFillPrice = selectDesiredTradeFillPrice(getState());
+		const { stopLossPrice, takeProfitPrice } = selectSlTpTradeInputs(getState());
+
+		try {
+			if (!marketInfo) throw new Error('Market info not found');
+			if (!account) throw new Error('No cross margin account found');
+			if (!wallet) throw new Error('No wallet connected');
+
+			dispatch(
+				setTransaction({
+					status: TransactionStatus.AwaitingExecution,
+					type: 'submit_cross_order',
+					hash: null,
+				})
+			);
+
+			const orderInputs: SmartMarginOrderInputs = {
```

Comment:
And then related to the other comment we can update this section to not include any other order details.